File: intro_to_ruby/vars/age.rb
puts "how old are you ? (please enter your age)"
age = gets.to_f # account for infants 0 < age < 1 yr
incr = 0
if age > 0
  age = age.to_i
  4.times do
    incr += 10
    puts "in #{incr} years you will be #{age + incr}"
  end
else
  puts "invalid age"
end

File: intro2js/exercises/io/age.js
// age.js

let rlSync = require('readline-sync');
let age = rlSync.question("What is your age ?\n");

if (age > 0) { // account for infants 0 < age < 1 yr
  age = parseInt(age, 10);
  for (let delta = 10; delta < 50; delta += 10) {
    console.log(`in ${delta} years you will be ${age + delta} years old.`);
  }
} else console.log('invalid age');
